.github/workflows/ghes-acceptance-tests-all.yml

```diff
@@ -0,0 +1,39 @@
+name: GHES Acceptance Tests (All)
+
+on:
+  schedule:
+    #         ┌───────────── minute (0 - 59)
+    #         │ ┌───────────── hour (0 - 23)
+    #         │ │ ┌───────────── day of the month (1 - 31)
+    #         │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #         │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #         │ │ │ │ │
+    #         │ │ │ │ │
+    #         │ │ │ │ │
+    #         * * * * *
+    - cron:  '*/15 * * * *'
+
+jobs:
+  acceptance-tests:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Acceptance Tests
+        uses: terraformtesting/acceptance-tests@v2.0.0
+        with:
+          RUN_ALL: true
+          GITHUB_BASE_URL: "https://terraformtesting-ghe.eastus.cloudapp.azure.com/"
+          GITHUB_OWNER: terraformtesting
+          GITHUB_TEST_USER: github-terraform-test-user
+          GITHUB_TEST_USER_NAME: "Test User"
+          GITHUB_TEST_USER_EMAIL: 60107403+github-terraform-test-user@users.noreply.github.com
+          GITHUB_TEST_USER_TOKEN: ${{ secrets.GITHUB_TEST_USER_TOKEN }}
```

Comment:
Doesn't the GHE endpoint require a separate token, i.e. different secret name?

Reply:
Yes, this will absolutely fail.  I have another PR behind this one to address correctness.  This PR lays a foundation from an Actions configuration perspective, but intends to be iterated on afterwards.